src/service-worker.ts

```diff
@@ -110,8 +111,10 @@ export async function generateServiceWorker(options: AddServiceWorkerOptions):
     return path.join(buildRoot, filePath);
   });
 
-  // swPrecache will determine the right urls by stripping buildRoot
-  swPrecacheConfig.stripPrefix = buildRoot;
+  // swPrecache will determine the right urls by stripping buildRoot.
+  // NOTE:(usergenic) sw-precache generate() apparently to replaces the
```

Comment:
I fixed the comment.  Thanks.

I'm not sure technically this counts as a *bug* with sw-precache, and of course if they fix it, it will break for people who are working around it by using forward slashes anyway.